ci-operator/step-registry/ipi/conf/libvirt/ipi-conf-libvirt-chain.yaml

```diff
@@ -0,0 +1,21 @@
+chain:
+  as: ipi-conf-libvirt
+  steps:
+#  - ref: ipi-conf
+  - ref: ipi-conf-libvirt
+  env:
+  - name: ARCH
+  - name: MASTER_REPLICAS
+  - name: NSS_GROUPNAME
+  - name: NSS_USERNAME
+  - name: NSS_WRAPPER_GROUP
+  - name: NSS_WRAPPER_PASSWD
+  - name: WORKER_REPLICAS
+  - name: REMOTE_LIBVIRT_HOSTNAME
+  - name: REMOTE_LIBVIRT_HOSTNAME_1
+  - name: REMOTE_LIBVIRT_HOSTNAME_2
+  - name: REMOTE_LIBVIRT_URI
+  documentation: |-
+    This chain generates an install-config.yaml file configured to run clusters in the libvirt CI project.
+    The libvirt specific configs are added to the file generated by the ipi-conf steps.
+    This resulting file is stored in the shared directory for future consumption.
```

Comment:
nit: mention `${SHARED_DIR}/install-config.yaml` to be explicit about where in the shared directory?  Compare with [this][1].

[1]: https://github.com/openshift/release/blob/3c8f4c10b7e2d2210e5bdf0eb334835714cfecf6/ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml#L24